exporter/containerimage/export.go

```diff
@@ -346,7 +348,15 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source,
 	resp[exptypes.ExporterImageDigestKey] = desc.Digest.String()
 	if v, ok := desc.Annotations[exptypes.ExporterConfigDigestKey]; ok {
 		resp[exptypes.ExporterImageConfigDigestKey] = v
+		delete(desc.Annotations, exptypes.ExporterConfigDigestKey)
```

Comment:
@tonistiigi Added the same removal for `config.digest` in `containerimage` exporter like it's done for the oci one: https://github.com/moby/buildkit/blob/a640b47cb19c4f0ff47f2444f3215ee851598a8e/exporter/oci/export.go#L214

Found out it was introduced in https://github.com/moby/buildkit/commit/1098f8ee82b8e1381e6866b253d8cb2b136ded79. Any reason to not remove the `config.digest` annotation for containerimage exporter?